### PR TITLE
data: jontstaz

### PR DESCRIPTION
**Pull request by jontstaz**
## Changes proposed

- Add my profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3250"></a>

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
File name does not match username

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Hi Jonte! The way linkfree is set up the name of your file needs to match your github username (case-sensitive). Once you've updated the name then this is good to merge.

**Comment by jontstaz** [comment]
Hi @eddiejaoude @EmmaDawsonDev - Sorry for the late reply but I've renamed the file to match my github username now and I've merged latest changes so its up-to-date w/ master. Should be all good to go now. Thx!

File: data/jontstaz.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Jonte",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm an API Solutions Engineer based in Melbourne with experience in API Development, DevSecOps, Cloud Security, and Algorithmic Trading Systems. Currently working as a Solutions Engineer for a major fintech company.",
+  "avatar": "http://github.com/jontstaz.png",
```
Comment (Panquesito7) [inline data/jontstaz.json:6]: The avatar field is no longer required as of a new update.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by jontstaz** [comment]
@Panquesito7 - Ok thanks for the info. I removed it as suggested now.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Comment by jontstaz** [comment]
@eddiejaoude - Mind approving the PR when you can so it can be merged?

**Review by eddiejaoude — APPROVED** [review]
Great collaboration everyone 👍

**Pull request merged**